src/duplicacy_backupmanager.go

```diff
@@ -1689,12 +1696,14 @@ func (manager *BackupManager) CopySnapshots(otherManager *BackupManager, snapsho
 
     for _, snapshot := range snapshots {
         if revisionMap[snapshot.ID][snapshot.Revision] == false {
-            continue
+          continue
+        }
+        if !manager.config.dryRun {
```

Comment:
CopySnapshots is used by the copy command only, so don't need to check the dry-run mode.